x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/cluster_details_box.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+  EuiToolTip,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import moment from 'moment';
+import React from 'react';
+import { INTERNAL_FEATURE_FLAGS } from '../../../../common/constants';
+import { Cluster } from '../../../../common/types';
+import { useNavigateFindings } from '../../../common/hooks/use_navigate_findings';
+import { CISBenchmarkIcon } from '../../../components/cis_benchmark_icon';
+
+export const ClusterDetailsBox = ({ cluster }: { cluster: Cluster }) => {
+  const navToFindings = useNavigateFindings();
+
+  const shortId = cluster.meta.clusterId.slice(0, 6);
+  const title = cluster.meta.clusterName || 'Cluster ID';
```

Comment:
Consider naming `Cluster ID` as a module-level constant

Reply:
yes and should probably be translated aswell